### DISCUSSION TITLE
Model Error during perfrom message_ix -> tutorial ->westeros_baseline

### DISCUSSION THREAD

**Discussion by JUSTSP2233** [iiasa/message_ix#638] [MESSAGE modeling] [answered]
I ran into this problem while dealing with the "Plotting Results" phase
in[30]
an error happen as below
![1659942934287](https://user-images.githubusercontent.com/46024213/183361575-5cf0ffae-568a-44fa-b98c-6a34d1bc8f94.png)
![1659942980791](https://user-images.githubusercontent.com/46024213/183361579-d5be0479-6e58-4feb-a13e-8d4ed4fe2ed9.png)
![1659943009264](https://user-images.githubusercontent.com/46024213/183361581-9b55efcc-1a42-4e7c-ae2b-bae41a04fcaf.png)
Hope to get your help,
Thank you.

**Comment by khaeru** [comment]
@JUSTSP2233, you haven't said what versions of the software you are using, e.g. the output of `message-ix show-versions`.

However, I suspect your issue would be resolved by upgrading to message_ix and ixmp versions 3.6.0. Can you please try that and report your results?

P.S. Please read the GitHub documentation about [how to paste code as text](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code), rather than a screenshot image.

**Comment by fatgroose** [comment, marked as answer]
you can upgrade the “ixmp” to the latest version for solving this problem.

**Comment by glatterf42** [comment]
Thank you for confirming that, @fatgroose. While I'm unable to say for certain, my guess is this was indeed fixed by khaeru/genno/pull/63 upstream, which was included in iiasa/ixmp/pull/451, and thus in version 3.6.0.

**Comment by fatgroose** [comment]
According to my experience，this problem still existed in version 3.6.0
before updating the lastest ixmp.🙂

Fridolin Glatter ***@***.***>于2023年4月26日 周三16:06写道：

**Comment by glatterf42** [comment]
Thanks for letting us know, though I have to admit that from looking at the code, I then have no idea which changes might have solved this issue. We are planning on releasing v3.7.0 soon; updating to the newest version will then make sense either way.
